controllers/imagerepository_controller_test.go

```diff
@@ -563,6 +565,33 @@ var _ = Describe("Image repository controller", func() {
 			imageRepository := getImageRepository(resourceKey)
 			Expect(imageRepository.Spec.Image.Name).To(Equal(expectedImageName))
 		})
+
+		It("should create image repository with requested name, includes already namespace", func() {
```

Comment:
`... with requested name that already includes ...`

Reply:
ok